Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3061,6 +3061,29 @@ extension BrowserViewController: PreferencesObserver {
             setupTabs()
             updateTabsBarVisibility()
             updateApplicationShortcuts()
+            
+            // Destroy all tab data
+            tabManager.allTabs.forEach({
```

Comment:
this first `(` bracket shouldn't be needed, can be removed

Reply:
I kept it there because the rest of the code has it. A search for `({` brings up 250+ results.